src/main/java/com/dabsquared/gitlabjenkins/gitlab/api/impl/V3GitLabClientBuilder.java

```diff
@@ -1,24 +1,33 @@
 package com.dabsquared.gitlabjenkins.gitlab.api.impl;
 
-import com.dabsquared.gitlabjenkins.gitlab.api.model.MergeRequest;
+import com.dabsquared.gitlabjenkins.gitlab.api.GitLabClientBuilder;
 import hudson.Extension;
-import java.util.function.Function;
+import org.gitlab4j.api.GitLabApi;
+import org.gitlab4j.api.GitLabApi.ApiVersion;
+import org.gitlab4j.api.GitLabApiException;
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
 
 @Extension
 @Restricted(NoExternalUse.class)
-public final class V3GitLabClientBuilder extends ResteasyGitLabClientBuilder {
-    private static final int ORDINAL = 2;
```

Comment:
Why remove this constant? If there is no reason to remove it then please keep it.